### PR TITLE
STORM-581. Add rebalance params to Storm REST API.

### PR DESCRIPTION
**Pull request by harshach**

### CODE REVIEW

**Comment by harshach** [comment]
@ptgoetz @revans2  Bobby already gave a +1 from previous PR. This new PR is an upmerge without any code changes. Can you please take a look.

File: STORM-UI-REST-API.md [hunk 1]
```diff
@@ -630,6 +642,31 @@ Rebalances a topology.
 |----------|--------|-------------|
 |id   	   |String (required)| Topology Id  |
 |wait-time |String (required)| Wait time before rebalance happens |
+|rebalanceOptions| Json (optional) | topology rebalance options |
+
+
+Sample rebalancOptions json:
```
Comment (revans2) [inline STORM-UI-REST-API.md]: really minor s/rebalancOptions/rebalanceOptions/ (missing an e)

**Comment by revans2** [comment]
One really minor spelling fix.  I am +1 for the code so unless the code changes you don't need another +1 from me.

**Comment by ptgoetz** [comment]
+1 The typo fix is a non-code change so I'm fine with including that.

**Comment by harshach** [comment]
fixed the doc. Thanks.

**Pull request merged**